EventListener/BugsnagListener.php

```diff
@@ -248,6 +248,9 @@ private function isOom($throwable)
             || $throwable instanceof OutOfMemoryException;
     }
 
+    /**
+     * @return array<string, array{string, int}>
```

Comment:
This is an array with string keys and values that are arrays where the 0th element is a string and 1st element is an int

This is one of the allowed formats in [the interface's `@return`](https://github.com/symfony/symfony/blob/80f4e7628511fafb38c5b2e4fbd85e264c247793/src/Symfony/Component/EventDispatcher/EventSubscriberInterface.php#L46)